test/lib/code_corps/user_task_matcher_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule CodeCorps.UserTaskMatcherTest do
+  use CodeCorps.ModelCase
+
+  import CodeCorps.UserTaskMatcher
+
+  test "can find top x tasks for a user's skill" do
+    coding = insert(:skill, title: "coding")
+    design = insert(:skill, title: "design")
+
+    account_page = insert(:task)
+    settings_page = insert(:task)
+    photoshop = insert(:task)
```

Comment:
That's what i was going for with skills that were implementing an entire page (thus needing `coding` and `design`) versus one that was just PhotoShop.

I did start to wonder sometimes about using just the most overlapping tasks as the sort priority. I could see someone like me selecting 'design' as a tag (even though that's a stretch). Then I'd get a list of results back that were design heavy, even though I'd be better suited for a task that simply overlaps my strongest skill, `coding`.

(Seems like this would be more clear when we saw real data instead of speculating what the data would be.)